staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -434,6 +439,13 @@ func NewCacherFromConfig(config Config) (*Cacher, error) {
 		)
 	}()
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.ConsistentListFromCache) {
+		supports := cacher.Supports(storage.RequestWatchProgress)
```

Comment:
Why remove error? We don't want merge etcd down with old etcd case.

Reply:
`Supports` only return from cache here. If we want to return error we either need to change `Supports` or cache the error.

Reply:
No, you call Storage.Supports() which calls etcdfeature.SupportsClient, which should return error.

Reply:
Updated! 
PTAL. Thanks.